File: Sellino.Portal/src/app/services/product/product.ts
export class CreateProductModel {
  Name: string;
  Description: string;
  Price: string;
  ProductGroupId: number;
  ProductMediaId: number;

  constructor(name: string, description: string, price: string, productGroupId: number, productMediaId: number) {
    this.Name = name;
    this.Description = description;
    this.Price = price;
    this.ProductGroupId = productGroupId;
    this.ProductMediaId = productMediaId;
  }
}

File: Sellino.Domain/Models/Product.cs
﻿using System.ComponentModel.DataAnnotations;

namespace Sellino.Domain.Models
{
    public class Product
    {
        [Key]
        public int ProductId { get; set; }
        public Guid ProductToken { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int ProductGroupId { get; set; }
        public int CreatedByUserId { get; set; }
        public int Price { get; set; }
        public int? ProductMediaId { get; set; }
        public DateTimeOffset DateCreated { get; set; }
        public DateTimeOffset DateModified { get; set; }
        public DateTimeOffset DateDeleted { get; set; }
        public bool IsDeleted { get; set; }
    }
}
